ui/admin/app/components/form/policy/policy-selection/index.js

```diff
@@ -15,39 +15,53 @@ export default class FormPolicySelectionComponent extends Component {
   get showCustomInput() {
     if (this.args.name === 'retention_policy') {
       return this.isCustomRetentionSelected;
-    } else if (
-      this.args.name === 'deletion_policy' &&
-      this.args.model.retain_for?.days >= 0
-    ) {
+    } else if (this.args.name === 'deletion_policy') {
       return this.isCustomDeletionSelected;
     } else {
       return null;
     }
```

Comment:
since `this.isCustomRetentionSelected` and `this.isCustomDeletionSelected` methods do the same thing now, i think this getter can be simplified and there is no need to separate it into two different methods. Also perhaps the else block on line 20 is not needed anymore ?

Reply:
💯 simplified!